docs/aegis/roadmap/core-tooling-compliance.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Core Tooling Compliance
+area: Compliance
+status: In Progress
+progress: 20
+funding_required: false
+sponsors:
+  - Ericsson
+  # TODO: Can we disclose the other sponsor?
+index: 4
+previous:
+  url: gleam-openchain-compliance
+  title: Gleam OpenChain Compliance
+next:
+  url: core-tooling-governance-audit
+  title: Core Tooling Governance Audit
+---
+
+## Goal
+
+Implement Compliance Frameworks for core tooling
+
+## Impact
+
+By establishing a robust compliance framework for core tooling used in projects
+like Erlang, Elixir, Gleam, and Hex, this milestone will drive the consistent
+adoption of industry-leading security and quality standards across the open
+source ecosystem. By integrating OpenSSF Best Practices Badge, Security
+Insights, and ScoreCard, developers can more easily identify, measure, and
+address critical security gaps, thus ensuring that each project aligns with
+recognized best practices. The addition of a statistics tool to track adoption
+further amplifies transparency, enabling real-time monitoring of improvements
+and facilitating continuous growth in community-wide security awareness.
+Ultimately, this milestone fosters a healthier and more resilient open source
+environment, elevating trust for developers and end-users alike.
+
+## Deliverables
+
+* Projects:
+  - erlang
+  - elixir
+  - gleam
+  - hex
+* Implement OpenSSF Best Practices Badge
+* Implement OpenSSF Security Insights
+* Implement Open Source Project Security Baseline
+* Implement & improve score for OpenSSF ScoreCard
```

Comment:
All of this is great, but I am unsure as to how much it impacts OTP and our resources.
If it requires more effort from Erlang/OTP than all the things we have done recently for scorecard, reuse, and ORT, we should discuss and plan it for one of our sprints (it can be done, just not immediately)

Reply:
@kikofernandez None of this is expected to be implemented immediately. But there's some things that could be improved without huge efforts (and I'd be happy to support as well).

For example: Differentiating Workflow / Job Permissions and Pinned Dependencies could be improved with minor amounts of work.

Reply:
Sounds good then!